pgbot/commands/base.py

```diff
@@ -645,10 +650,19 @@ async def handle_cmd(self):
             )
         except discord.NotFound:
             # response message was deleted, send a new message
-            await embed_utils.send(
+            target_message = await embed_utils.send(
                 self.channel,
                 title=title,
                 description=msg,
                 color=0xFF0000,
                 footer_text=excname,
             )
+
+        task = asyncio.create_task(
+            message_delete_reaction_listener(target_message, self.author, emoji_str="🗑")
+        )
+
+        try:
+            await task
```

Comment:
why not a simple `await message_delete_reaction_listener(target_message, self.author, emoji_str="🗑")` here? If doing things this way gives different behvaviour, i should be commented IG

Reply:
good point

Reply:
I've resolved this part.